templates/postfix/main.cf.j2

```diff
@@ -110,6 +115,8 @@ smtp_tls_security_level = may
 smtp_dns_support_level = dnssec
 smtp_tls_security_level = dane
 {% endif %}
+smtpd_tls_loglevel = 1
```

Comment:
How noisy is this?

Reply:
For each SMTP connection, you get one log entry like
```
Dec  1 12:11:39 core-fal.njsm.de postfix/submission/smtpd[14303]: Anonymous TLS connection established from nfw2.sos.ethz.ch[192.33.91.239]: TLSv1.2 with cipher ECDHE-ECDSA-AES256-GCM-SHA384 (256/256 bits)
```
or
```
Dec  1 12:36:43 core-fal.njsm.de postfix/smtp[2406]: Verified TLS connection established to mx2.mailbox.org[80.241.60.215]:25: TLSv1.2 with cipher ECDHE-RSA-CHACHA20-POLY1305 (256/256 bits)
```
which is the only way to see if DANE was used.